modules/platforms/dotnet/Apache.Ignite.Benchmarks/Table/TupleGetBenchmarks.cs

```diff
@@ -23,11 +23,12 @@ namespace Apache.Ignite.Benchmarks.Table
     using Tests;
 
     /// <summary>
-    /// Results on Intel Core i7-7700HQ, .NET SDK 5.0.400, Ubuntu 20.04:
-    /// | Method |     Mean |   Error |  StdDev |
-    /// |------- |---------:|--------:|--------:|
-    /// |    Get | 202.0 us | 3.71 us | 7.82 us |.
+    /// Results on i9-12900H, .NET SDK 6.0.405, Ubuntu 22.04:
+    /// | Method |     Mean |     Error |    StdDev |
+    /// |------- |---------:|----------:|----------:|
+    /// |    Get | 4.788 ms | 0.3198 ms | 0.0830 ms |.
```

Comment:
Have it became slower? Or does "ms" stands for "microseconds"? Probably, "us" should be used in this case.

Reply:
Yes, it has become slower. Previous result is from over a year ago. Units are correct, those are milliseconds.

Reply:
I've updated the benchmark to compare this branch with current master. But we can't see the difference with results like those, and such a deviation.